components/Styled/IconButtonLarge.js

```diff
@@ -0,0 +1,42 @@
+import styled from "styled-components";
+import ArrowLeft from "@/public/icons/ArrowSmall.svg";
+import ChevronSmall from "@/public/icons/ChevronSmall.svg";
+import Plus from "@/public/icons/Plus.svg";
+import XSmall from "@/public/icons/XSmall.svg";
+
+export default function IconButtonLarge({ onClick, style }) {
+  const buttonStyles = {
+    arrowLeft: <ArrowLeft width={30} height={30} />,
+    chevrondown: <ChevronSmall width={30} height={30} />,
+    plus: <Plus width={30} height={30} />,
+    x: <XSmall width={30} height={30} />,
```

Comment:
I think you could implement this more beautifully with the styled components props :)
https://styled-components.com/docs/basics#adapting-based-on-props

Reply:
thank you very much for this hint! After merging this and the hasCooked feature we will then refacture this part all together and use the styled components probs consistently:)